util/Cargo.toml

```diff
@@ -24,6 +24,9 @@ thiserror = { workspace = true }
 mio = { workspace = true }
 tempfile = { workspace = true }
 uds = { workspace = true, optional = true, features = ["mio_1xx"] }
+libcrux-test-utils = { workspace = true, optional = true }
+lazy_static = { workspace = true, optional = true }
```

Comment:
I thought lazy_static was now mostly superfluous and we can use standard library features instead (OnceLock)

Reply:
It is with newer versions of Rust, but not with your MSRV

Reply:
OnceLock is 1.70 

https://doc.rust-lang.org/std/sync/struct.OnceLock.html

LazyLock is 1.80

https://doc.rust-lang.org/std/sync/struct.LazyLock.html

would it add significant complexity to build this using OnceLock? If its easy, we should replace it IMO, if its hard, adding the dependency is well-justified :)